services/treeware/treeware-trees.service.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const crypto = require('crypto')
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { floorCount } = require('../color-formatters')
+const { BaseJsonService } = require('..')
+
+const apiSchema = Joi.object({
+  total: Joi.number().required(),
+}).required()
+
+module.exports = class TreewareTrees extends BaseJsonService {
+  static get category() {
+    return 'other'
+  }
+
+  static get route() {
+    return {
+      base: 'treeware/trees',
+      pattern: ':owner/:packageName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Treeware (Trees)',
+        namedParams: { owner: 'stoplightio', packageName: 'spectral' },
+        staticPreview: this.render({ count: 250 }),
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'trees' }
+  }
+
+  static render({ count }) {
+    return { message: metric(count), color: floorCount(count, 10, 50, 100) }
+  }
+
+  async fetch({ reference }) {
+    const url = `https://public.offset.earth/users/treeware/trees?ref=${reference}`
+    return this._requestJson({
+      url,
+      schema: apiSchema,
+      errorMessages: {
+        404: 'repository not found',
```

Comment:
I don't think there are any known cases where the API returns a 404 (i.e. using a random owner and package name return 0 trees): we can probably remove this `errorMessages`parameter.